src/db/mod.rs

```diff
@@ -425,7 +425,7 @@ impl ThreadedDatabase for Database {
     fn get_user(&self, user: &str, options: Option<UserInfoOptions>) -> Result<bson::Document> {
         let mut doc =
             doc! {
-            "usersInfo" => { "user" => user, "db" => (self.name.to_owned()) }
+            "usersInfo": { "user": user, "db": self.name.to_owned() }
```

Comment:
For multiple keys, I think it's probably cleaner to have things on multiple lines. (rustfmt will still probably be weird about the indentation)